crypto/encoding/amino/amino.go

```diff
@@ -62,8 +62,11 @@ func RegisterAmino(cdc *amino.Codec) {
 
 // RegisterKeyType registers an external key type to allow decoding it from bytes
 func RegisterKeyType(o interface{}, name string) {
-	cdc.RegisterConcrete(o, name, nil)
+	if _, exists := nameTable[reflect.TypeOf(o)]; exists {
+		return
```

Comment:
I think we should panic, not silently return

Reply:
Yeah, I can do that, but isn't that basically the same panic that occurs in `RegisterConcrete` when you try to register an existing type?

Reply:
Did not know we panic in Register concrete. If that's the case, we can get rid of the extra check. Thank you